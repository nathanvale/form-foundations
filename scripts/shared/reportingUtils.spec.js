'use strict';

const reportingUtils = require('./reportingUtils');
const { printToTerminalWidth, printInverseMessage, FAIL, OK } = reportingUtils;

describe('FAIL', () => {
  it('should...', () => {
    const actual = {};
    expect(FAIL).toMatchSnapshot();
  });
});

describe('OK', () => {
  it('should...', () => {
    const actual = {};
    expect(OK).toMatchSnapshot();
  });
});
