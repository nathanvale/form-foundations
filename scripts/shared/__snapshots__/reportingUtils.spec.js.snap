// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FAIL should... 1`] = `undefined`;

exports[`OK should... 1`] = `undefined`;
